.editorconfig

```diff
@@ -0,0 +1,19 @@
+root = true
+
```

Comment:
Yes please that'd be appreciated. As I test this out today, if you like I can tackle any issues that run up. That way I don't keep bothering about little stuff like that.